config/redirects

```diff
@@ -4,29 +4,33 @@ define: versions master
 
 # raw: <source file> -> ${base}/<destination>
 
-# Redirect calculated fields to new drawer introduced in DOCSP-32370 - Field Customizations
-# Field Customizations begin
+# DOCSP-32370 - Field Customizations moved under new drawer page
 [*]: ${prefix}/mapping-rules/calculated-fields/calculated-fields -> ${base}/mapping-rules/fields/calculated-fields/calculated-fields/
 [*]: ${prefix}/mapping-rules/calculated-fields/calculated-fields/add-calculated-fields -> ${base}/mapping-rules/fields/calculated-fields/calculated-fields/add-calculated-fields
 [*]: ${prefix}/mapping-rules/calculated-fields/calculated-fields/view-calculated-fields -> ${base}/mapping-rules/fields/calculated-fields/calculated-fields/view-calculated-fields
 [*]: ${prefix}/mapping-rules/calculated-fields/calculated-fields/edit-calculated-fields -> ${base}/mapping-rules/fields/calculated-fields/calculated-fields/edit-calculated-fields
 [*]: ${prefix}/mapping-rules/calculated-fields/calculated-fields/delete-calculated-fields -> ${base}/mapping-rules/fields/calculated-fields/calculated-fields/delete-calculated-fields
-# Field Customizations end
 
-# Redirect for previous Code Generation Page DOCSP-33055 - Prep Code Generation
-# Prep Code Generation begin
+# DOCSP-33055 - Prep Code Generation
 [*]: ${prefix}/code-generation/code-generation/ -> ${base}/code-generation/
-# Prep Code Generation end
 
-# DOCS-16498 Getting Started Page Start
+# DOCS-16498 Getting Started page
 [*]: ${prefix}/migration-scenarios/ -> ${base}/getting-started/migration-scenarios/
 [*]: ${prefix}/supported-databases/ -> ${base}/getting-started/supported-databases/
-# DOCS-16498 Getting Started Page End
 
 # DOCSP-46666 repetitive paths
 [*]: ${prefix}/projects/projects/ -> ${base}/projects/
 [*]: ${prefix}/table-filters/table-filters/ -> ${base}/table-filters/
 
+# DOCSP-46665 repetitive paths
+[*]: ${prefix}/mapping-rules/mapping-rule-options/mapping-rule-options/ -> ${base}/mapping-rules/mapping-rule-options/introduction/
+[*]: ${prefix}/mapping-rules/mapping-rules/ -> ${base}/mapping-rules/introduction/
+
+# DOCSP-46664 repetitive paths
+[*]: ${prefix}/jobs/data-verification/data-verification/ -> ${base}/jobs/data-verification/
+[*]: ${prefix}/mapping-rules/fields/calculated-fields/calculated-fields/ ->
+${base}/mapping-rules/fields/calculated-fields/
```

Comment:
Moved these up to keep things in ticket order

Reply:
Is this the order they were solved in? Thanks!

Reply:
It's not, but the tickets were all done in a span of a couple of days. I thought it was more valuable for readability to quickly see the ticket list and confirm there weren't gaps. Do you think I should revert the order change?

Reply:
It's not clear to me what the order is we're preserving, that's really all. I think it makes sense to have them fairly centralized.